### ISSUE TITLE
Another PSD that causes a crash on read

### ISSUE THREAD

**Issue by Pizzacus** [Agamnentzar/ag-psd#75]
Hi it's me again

I've found another PSD that crashes when read, this one does actually work fine in GIMP and Krita: https://cdn.discordapp.com/attachments/345312747631673346/862390763186814976/001.psd

The following error appears:

```
Error: Invalid signature: 'MeSa' at 0x7354
    at checkSignature (.../node_modules/.pnpm/ag-psd@14.0.2/node_modules/ag-psd/dist/psdReader.js:136:15)
    at _loop_1 (.../node_modules/.pnpm/ag-psd@14.0.2/node_modules/ag-psd/dist/psdReader.js:171:13)
    at .../node_modules/.pnpm/ag-psd@14.0.2/node_modules/ag-psd/dist/psdReader.js:197:13
    at readSection (.../node_modules/.pnpm/ag-psd@14.0.2/node_modules/ag-psd/dist/psdReader.js:640:18)
    at Object.readPsd (.../node_modules/.pnpm/ag-psd@14.0.2/node_modules/ag-psd/dist/psdReader.js:169:5)
    at Object.readPsd (.../node_modules/.pnpm/ag-psd@14.0.2/node_modules/ag-psd/dist/index.js:26:24)
    at parseFile (.../dist/index.js:61:32)
```

I'm on Node v16.3.0 and using the latest version 14.0.2 of the lib on Linux. I read with `{ useImageData: true }`

Again, this is nothing important, don't feel urged to work on it if you'd rather do something else.

Thanks!

**Comment by Agamnentzar** [comment]
Should be fixed in version `14.1.0`, always good to fix crashing issues.

**Comment by Pizzacus** [comment]
It works! Thanks again for the quick fix :heart:
